gitpod-io/gitpod
# [ws-daemon] Restart IWS if ws-daemon is restarted

## Description


## Related Issue(s)

Fixes #

## How to test

1. start an image build and while the image build is running delete the ws-daemon pod
2. start a workspace, once it's running delete the ws-daemon pod. At any point `docker run --rm  -t alpine:latest date ` should work

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - fo-ws-daemon-restart</li>
	<li><b>🔗 URL</b> - <a href="https://fo-ws-daemon-restart.preview.gitpod-dev.com/workspaces" target="_blank">fo-ws-daemon-restart.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
